src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { Todo } from '../../Types/Todo';
+import usersFromServe from '../../api/users';
+import { getUserById } from '../../services/user';
+import { getNewTodoId } from '../../services/id';
```

Comment:
The functions `getUserById` and `getNewTodoId` should not be used directly in the `TodoForm` component. Instead, manage these operations in the `App` component and pass the necessary data or functions as props to `TodoForm`.